rules/X++/MissingMethodDocumentation.xq

```diff
@@ -12,13 +12,12 @@ declare function functx:trim ( $arg as xs:string? )  as xs:string
 {
   for $c in /(Class | Table | Form)[@IsPrivate = "false"]
   for $m in $c//Method[@IsPrivate = "false" and functx:trim(@Comments) = ""]
```

Comment:
I think comments can also come out as `"(null)"`. Let me validate this and come back to you.

Reply:
Nope, never mind 😄.